04/index.html

```diff
@@ -5,8 +5,16 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>devmentor.pl - HTML & CSS: RWD - #04</title>
+    <link rel="stylesheet" href="global.css">
+    <link rel="stylesheet" media="(min-width: 600px)" href="normal.css">
 </head>
 <body>
+    <label for="menu-switcher">
+        <button class="hamburger"></button>
+    </label>
```

Comment:
Żeby opcja z `:focus` zadziałałą to trzeba zmienić HTML na coś takiego:
```
    <label for="menu-switcher">

    </label>
   
    <button id="menu-switcher" class="hamburger"></button>
```

Ta zmiana jest niezbędna bo to `button` jest aktywny. W CSS nie mamy jak się cofnąć, aby ostylować przy pomocy selektrora `nav` dlatego zmieniamy strukturę. Teraz nav oraz button to rodzieństwo i zadziałą selektro `+`.

Natomiast klikanie w label nadal działa bo ten element jest połączony z buttonem przez for="menu-switcher" -> id="menu-switcher".

Mam nadzieje, że teraz rozjaśniłem wszystko :)